shared/menubar/index.render.desktop.js

```diff
@@ -53,13 +53,16 @@ class Render extends Component<void, Props, State> {
   }
 
   _menuItems () {
-    return [
-      {title: 'Open folders', onClick: this.props.showKBFS},
-      {title: 'Keybase.io', onClick: this.props.showUser},
-      {title: 'Report a bug', onClick: this.props.showBug},
-      {title: 'Help/Doc', onClick: this.props.showHelp},
-      {title: 'Quit', onClick: this.props.quit}
-    ]
+    return [].concat(
+      this.props.loggedIn ? [{title: 'Open Keybase', onClick: this.props.openApp}] : [],
+      [
+        {title: 'Open folders', onClick: this.props.showKBFS},
```

Comment:
yeah, takes you to /keybase. same as logged in. in the previous version we actually showed the public folders part when you're logged out. you still have access to all public shares. kinda in a wonky state now but let's keep it for now